charts/monochart/templates/deployment.yaml

```diff
@@ -153,10 +153,7 @@ spec:
         {{- end }}
         {{- if .Values.persistence.enabled }}
         - mountPath: {{ .Values.persistence.mountPath | quote }}
-          name: storage
-        {{- else if .Values.deployment.ephemeral.enabled }}
-        - mountPath: {{ .Values.deployment.ephemeral.mountPath | quote }}
-          name: scratch
+          name: {{ .Values.persistence.storageName }}
```

Comment:
I reaslized the storageName attribute defined in values.yaml wasn't used so I just made use of it.